app/views/practices/implementation_forms/_resources.html.erb

```diff
@@ -0,0 +1,129 @@
+            %>
+            <% if show_form %>
+            <% if pr.object.id %>
+              <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+                $(document).arrive('<%= "#practice_resources_attributes_#{pr_id}_resource" %>',
+                function(newElem) {
+                styleOriginFacility(
+                $(newElem).closest('li'),
+                <%= pr_id %>,
+                '.<%= slug %>-li',
+                '.<%= slug %>-ul',
+                '.<%= add_practice_resources_link %>>',
+                '12'
+                );
+                $(document).unbindArrive('<%= "#practice_resources_attributes_#{pr_id}_resource_type" %>', newElem);
+                }
+                );
+              <% end %>
+            <% end %>
+
+            <li class="<%= li_class_place_holder %>-li fields margin-bottom-2" data-id="<%= pr_id %>">
+              <%= pr.hidden_field :position %>
+              <%= pr.hidden_field :id %>
+              <div class="grid-row">
+                <div class="grid-col-12">
+                  <%= pr.label :resource, 'Resource', class: 'usa-label display-inline-block practice-resources-duration' %>&nbsp;
+                  <%= pr.text_field :resource, class: 'usa-input practice-input timeline-input grid-col-12', required: true %>
+                  <%= pr.hidden_field :resource_type_label, value: pr.object.id ? resource_type_label : resource_type_label_place_holder %>
+                  <%= pr.hidden_field :resource_type, value: pr.object.id ? resource_type : resource_type_place_holder %>
+                  <%= pr.hidden_field :media_type, value: pr.object.id ? media_type : media_type_place_holder %>
+                </div>
+                <div class="grid-col-12 trash-container">
+                  <%= pr.link_to_remove 'Delete entry',
+                                        class: "dm-origin-trash font-sans-xs line-height-26 cursor-pointer float-right"
+                  %>
+                </div>
+              </div>
+            </li>
+          <% end %>
+        <% end %>
+        </ul>
+        <br />
+        <% cur_resources = @practice.practice_resources.where(resource_type_label: resource_type_label, resource_type: resource_type, media_type: media_type) %>
+        <%= f.link_to_add :practice_resources, id: link_to_add_link_slug, 'data-dm-blueprint-id': blue_print_id, class: add_practice_resources_link + " dm-add-another-link line-height-26 cursor-pointer #{'display-none' unless cur_resources.any?}" do %>
+          <span class="font-sans-xs text-normal">Add another</span>
+        <% end %>
+        <%= f.link_to_add :practice_resources, id: link_to_add_button_slug, 'data-dm-blueprint-id': blue_print_id, class: add_practice_resources_link + " usa-button--outline dm-btn-primary #{'display-none' if cur_resources.any?}" do %>
+          <span class="padding-left-1 font-sans-md text-normal text-no-underline">Add resource</span>
```

Comment:
Remove the left padding and make font weight bold.

Reply:
Done.
![image](https://user-images.githubusercontent.com/60527788/94870285-04acaf80-03fc-11eb-9ec8-45a522e53ea7.png)